fe/fe-core/src/main/java/com/starrocks/sql/optimizer/statistics/ColumnHistogramStatsCacheLoader.java

```diff
@@ -154,8 +161,12 @@ private List<Bucket> convertBuckets(String histogramString, Type type) {
 
     private Map<Double, Long> convertTopN(String histogramString, Type type) {
```

Comment:
If you want to use `mcv` to replace `TopN`, please change all places.

Reply:
done